src/components/Highlight.vue

```diff
@@ -1,5 +1,4 @@
-<script>
-import { HIGHLIGHT_PRE_TAG, HIGHLIGHT_POST_TAG } from '../store';
+<script>import { HIGHLIGHT_PRE_TAG, HIGHLIGHT_POST_TAG } from '../store';
```

Comment:
ok so prettier does that automatically, no way I can fix it for now (maybe a vue plugin would work better than the html  one)